syntax = "proto3";

import "google/protobuf/timestamp.proto";

message User {
  string user_id = 1;
  bytes labels = 2;
  string comment = 3;
  repeated string subscribe = 4;
}

message Item {
  string namespace = 1;
  string item_id = 2;
  bool is_hidden = 3;
  repeated string categories = 4;
  google.protobuf.Timestamp timestamp = 5;
  bytes labels = 6;
  string comment = 7;
}

message Feedback {
  string namespace = 1;
  string feedback_type = 2;
  string user_id = 3;
  string item_id = 4;
  double value = 5;
  google.protobuf.Timestamp timestamp = 6;
  string comment = 7;
}
